docs/manual/templates.rst

```diff
@@ -144,16 +144,52 @@ Expressie                           Voorbeeld waarde                   Toelichti
 ``{{ now|date:"Y" }}``              ``2022``                           Huidig jaar
 ``{{ now|date:"F" }}``              ``augustus``                       Huidige maandnaam
 ``{{ now|date:"l" }}``              ``dinsdag``                        Huidige dagnaam
+``{{ now|date:"W" }}``              ``34``                             ISO-8601 weeknummer
 ``{{ now|time:"H:i" }}``            ``09:08``                          Huidig tijstip (uren en minuten)
 ``{{ now|time:"H:i:s" }}``          ``09:08:42``                       Huidig tijstip (uren, minuten en seconden)
-``{{ now|date:"W" }}``              ``34``                             ISO-8601 weeknummer
 ``{{ legeVariabele|default:"-" }}`` ``-``                              Terugvalwaarde indien de variabele "leeg" is
 ``{{ filesize|filesizeformat }}``   ``117,7 MB``                       Weergave van bytes (nummer) in leesbare vorm
 ``{{ consent|yesno:"ok,niet ok"}}`` ``niet ok``                        Weergave op basis van ``True``/ ``False`` waarde
+``{{ getal|add:"2" }}``             ``5``                              Equivalent van de som ``getal + 2``
+``{{ getal|add:"-2" }}``            ``1``                              Verminder de variabele ``getal`` met 2
+``{{ getal|floatformat }}``         ``3,1``                            Rond een getal af op één decimaal als er een decimaal gedeelte is
+``{{ getal|floatformat }}``         ``3``                              Indien er geen decimaal gedeelte is, toon dan geen decimalen
+``{{ getal|floatformat:"2" }}``     ``3,00``                           Rond altijd het getal af op twee decimalen
+``{{ getal|floatformat:"-2" }}``    ``3``                              Rond het getal af op twee decimalen als er een decimaal gedeelte is
+``{{ getal|floatformat:"2g" }}``    ``3.000,00``                       De ``g`` suffix past groepering toe
+``{{ getal|stringformat:"i" }}``    ``2023``                           Geef
+``{{ lijst|join:", " }}``           ``a, b, c``                        Voeg elementen in een lijst van waarden samen, gescheiden door een komma
+``{{ variabele|length }}``          ``12``                             Berekend de lengte van een lijst of string
```

Comment:
```suggestion
``{{ variabele|length }}``          ``12``                             Bereken de lengte van een lijst of string
```
or with `t`? I see we alternate between the two forms in the other descriptions

Reply:
for your Dutch language skills purposes: should be `t` instead of `d` indeed :sweat_smile: